Doc/library/http.client.rst

```diff
@@ -497,7 +497,8 @@ Here is an example session that uses the ``GET`` method::
    b'<!doctype html>\n<!--[if"...
    ...
    >>> # Example of an invalid request
-   >>> conn.request("GET", "/parrot.spam")
+   >>> conn = http.client.HTTPSConnection("httpbin.org")
```

Comment:
I would suggest hitting a python host that is going to be up for the forseeable future. `docs.python.org` for example returns the "correct" message of `404 Not Found`.

Reply:
Thanks, I have changed the the host to `docs.python.org` and tested the same.